dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/tei/query/ProgramAttributeCondition.java

```diff
@@ -40,6 +40,7 @@
 import org.hisp.dhis.analytics.shared.query.BinaryConditionRenderer;
 import org.hisp.dhis.analytics.shared.query.Field;
 import org.hisp.dhis.analytics.shared.query.OrCondition;
+import org.hisp.dhis.analytics.tei.query.context.QueryContext;
```

Comment:
I'm not sure if we need another package-level "context". We are going to seven levels...